includes/wsuwp-extended-customizer-capabilities.php

```diff
@@ -1,12 +1,13 @@
 <?php
 
-namespace WSUWP\Customizer_Capabilities;
-
-add_action( 'after_setup_theme', 'WSUWP\Customizer_Capabilities\bootstrap' );
-/**
- * Starts things up.
- *
- * @since 0.0.1
- */
-function bootstrap() {
+namespace WSU\Customizer_Capabilities;
+
+add_filter( 'map_meta_cap', 'WSU\Customizer_Capabilities\add_customizer_to_editors', 10, 3 );
+
+function add_customizer_to_editors( $caps, $cap, $user_id ) {
+	if ( 'customize' === $cap && user_can( $user_id, 'publish_pages' ) ) {
+		$caps = array( 'publish_pages' );
```

Comment:
Wait - is this meant to be `customize`?

Reply:
Nope, `publish_pages` is right. The `customize` cap is mapped to `edit_theme_options`, so this replaces that. I'm no good with meta caps, so I'm not entirely sure why it works. :)

Reply:
👍  It's making more sense to me this morning :)